content/docs/learning-guide/tutorials/animation/data-driven-root-motion.md

```diff
@@ -0,0 +1,152 @@
+
+{{< note >}}
+The preceding example movies display the actor's bones as green lines. The green line that extends from the ground plane to the actor's pelvis is the root bone of the skeleton. In an ideal scenario, actor assets should have this root bone, and any animation that moves the actor off of the scene origin should be applied to this root bone.
+{{< /note >}}
+
+### Which should you use?
+
+Whether you use code driven or data driven root motion depends on your project's needs.
+
+Code driven root motion might be preferable in scenarios where precise actor movement for play mechanics is more critical than visual performance. Platform games and fighting games, for example, usually require accurate actor movement. Ensuring an actor moves a precise distance in response to an input might be far more important than ensuring actor's movement relative to animation is visually accurate.
+
+Data driven root motion can give animators more control over the actor's performance. Adventure games and first person games that strive for realism and immersion often require the actor's movement express some game condition or emotion. In these scenarios, the actor's movement relative to animation might prioritize visual accuracy.
+
+The remainder of this tutorial focuses on data driven root motion. You'll learn how to determine if an actor is properly set up for data driven root motion, how to extract root motion if necessary, and how to enable root motion in [Animation Editor](/docs/user-guide/visualization/animation/animation-editor/user-interface).
+
+## Root motion set up
+
+The data in data driven root motion is usually keyframe animation applied to the root bone of an actor's skeleton by an animator. In an ideal scenario, the actor and animation source scene files follow these rules:
+
+* The skeleton has a root bone placed at the scene origin.
+* The root bone is connected to the actor's pelvis bone (or some relative bone on an actor that isn't a biped).
+* Any motion that moves the actor off the scene origin is keyframed on the root bone.
+
+### Check actor set up
+
+To determine if the actor has a root bone with animation, do the following:
+
+1. In **O3DE Editor**, from the **Tools** menu, choose **Animation Editor**.
+
+2. In Animation Editor, press **Ctrl + O** to open the actor. Select an actor from the file window.
+
+3. Choose the {{< icon "visibility-on.svg" >}} **Visibility** button above the viewport, and select **Line skeleton** from the list to display the actor's skeleton as green lines.
+
+4. In the **Motion Sets** tab, in the **Motion Set** group, choose the {{< icon "file-folder.svg" >}} **Folder** button to add a motion. Select a motion from the file window.
+
+5. Choose the motion you just added to the Motion Set to enable it.
+
+6. In the **Time View** tab, click the {{< icon "play.svg" >}} **Play** button to play the animation.
```

Comment:
uber nit: It might be easier to just update all these to `1.` as the markdown viewer will the automatically update the numbers if you ever add anything in the middle in future for example, makes maintenance much easier 🙂

Reply:
Yes, I use the 1. notation when I write but there's some setting in my editor that keeps converting them.

Reply:
Ah okay cool np :+1:

Reply:
fixed